app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -41,12 +45,19 @@
 
       <% if sl_helper.feed_link_box_value || sl_helper.feed_link %>
         <li class="gem-c-subscription-links__list-item<%= ' gem-c-subscription-links__list-item--small' if local_assigns[:small_form] == true %>">
-          <svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" class="gem-c-subscription-links__icon" focusable="false" fill="currentColor" aria-hidden="true"><path d="M1.996 11A2 2 0 0 0 0 12.993c0 1.101.895 1.99 1.996 1.99 1.106 0 2-.889 2-1.99a2 2 0 0 0-2-1.993zM.002 5.097V7.97c1.872 0 3.632.733 4.958 2.059A6.984 6.984 0 0 1 7.015 15h2.888c0-5.461-4.443-9.903-9.9-9.903zM.006 0v2.876c6.676 0 12.11 5.44 12.11 12.124H15C15 6.731 8.273 0 .006 0z"/></svg>
-          <%= link_to sl_helper.feed_link_text, sl_helper.feed_link,
+          <% feed_icon = capture do %>
+            <svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" class="gem-c-subscription-links__icon" focusable="false" fill="currentColor" aria-hidden="true"><path d="M1.996 11A2 2 0 0 0 0 12.993c0 1.101.895 1.99 1.996 1.99 1.106 0 2-.889 2-1.99a2 2 0 0 0-2-1.993zM.002 5.097V7.97c1.872 0 3.632.733 4.958 2.059A6.984 6.984 0 0 1 7.015 15h2.888c0-5.461-4.443-9.903-9.9-9.903zM.006 0v2.876c6.676 0 12.11 5.44 12.11 12.124H15C15 6.731 8.273 0 .006 0z"/></svg>
+          <% end %>
+          <% feed_link_text = capture do %>
+           <%= feed_icon %>
+           <%= sl_helper.feed_link_text %>
+          <% end %>
+          <%= link_to feed_link_text, sl_helper.feed_link,
```

Comment:
Open to discussion on this one - when this link is showing and hiding the feed link, should this actually be a button element? 

The [GDS Way recommends using a button rather than an empty link](https://gds-way.cloudapps.digital/manuals/programming-languages/html.html#buttons-vs-links) - but I appreciate that this is a bit of a strange situation.

The [tag helper](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag) might help prevent this going into if soup.

Reply:
Hey @injms that's a good point.
My rationale in leaving it as a link was to preserve the original functionality, where when JS was disabled, it would degrade gracefully to link straight to the feed URL. 
I suppose it does not _have_ to be this way.  As the component gracefully degrades to having the feed box visible with JS disabled. 
Based on the page you linked in your comment it does seem like it would be most appropriate to have it as a button instead so I will explore that and get back to you.

Reply:
@injms  Update: I've changed it to a button